products/goland.md

```diff
@@ -0,0 +1,44 @@
+---
+title: GoLand Plugin Development
+---
+
+## Introduction
+GoLand is an IntelliJ Platform-based product.
+Plugin projects for GoLand can be developed using IntelliJ IDEA with the `gradle-intellij-plugin`.
+
+## Configuring Plugin Projects Targeting GoLand
+The configuration of GoLand plugin projects follows the methods described in [Configuring Plugin Projects using the IntelliJ IDEA Product Attribute](dev_alternate_products.md#configuring-plugin-projects-using-the-intellij-idea-product-attribute), and [Configuring the plugin.xml File](dev_alternate_products.md#configuring-pluginxml).
+
+The table below summarizes the `gradle-intellij-plugin` attributes to set in the `build.gradle` file for a GoLand plugin project:
+
+| `gradle-intellij-plugin` Attribute | <br>Attribute Value |
```

Comment:
"`<br>`" ?

Reply:
Just formatting.  Stretch the browser window very wide, then also very narrow. It keeps the headings nearest the table contents.